showcase/src/assets/translations/en.json

```diff
@@ -58,6 +58,23 @@
         "SQL": "Pretty-Print SQL",
         "TYPESCRIPT": "Pretty-Print Typescript",
         "XML": "Pretty-Print XML"
+      },
+      "DROPDOWNS": {
+        "PR": "IT applications",
+        "IO": "Informatics infrastructure and operations",
+        "CS": "IT customer services",
+        "REQUIRED": "Single required selection",
+        "BLANK": "Single selection with default blank",
+        "MULTIPLE": "Multiple selection",
+        "MULTIPLE_REQUIRED": "Required multiple selection",
+        "DISABLED": "Disabled dropdown",
+        "SELECTED_VALUE": "Selected value: ",
```

Comment:
Actually, I find weird having ": " in our translations...
Could you maybe move those characters in the HTML directly ? 😊